libnmstate/nm/plugin.py

```diff
@@ -104,7 +103,6 @@ def plugin_capabilities(self):
         return [
             NmstatePlugin.PLUGIN_CAPABILITY_IFACE,
             NmstatePlugin.PLUGIN_CAPABILITY_ROUTE,
-            NmstatePlugin.PLUGIN_CAPABILITY_ROUTE_RULE,
```

Comment:
The NM plugin still have power on modifying the route rule.

The better approach might be:
 * Allowing any plugin with `NmstatePlugin.PLUGIN_CAPABILITY_ROUTE_RULE` to provide route rule, and merged them before showing. Please check `_get_routes_from_plugins()` as example.

Reply:
Done! Thank you!